backend/sql/connection.js

```diff
@@ -1,14 +1,40 @@
 const mysql = require('mysql');
+const util = require('util');
 
 const db_config = {
+	//DEV database
 	connectionLimit: 100,
 	host: 'us-cdbr-iron-east-03.cleardb.net',
 	user: 'bd17b1984d3b71',
 	password: 'cc800052',
 	connectionTimeout: 10000,
-	database: 'heroku_f11d5f6ef0dc063'
+	database: 'heroku_f11d5f6ef0dc063',
+	multipleStatements: true
+
+	//PROD database 
+	/*
+	connectionLimit: 100,
+	host: 'us-cdbr-iron-east-03.cleardb.net',
+	user: 'bc3b1bec1b7226',
+	password: 'b6054e24',
```

Comment:
I don't think the production database credentials should have ended up on Github.

Reply:
@MartyO256 Thanks for that, I commented this in another pull request. I appreciate the contribution! How did you find yourself here lol